docs/reference/dlm/index.asciidoc

```diff
@@ -0,0 +1,58 @@
+[role="xpack"]
+[[data-lifecycle-management]]
+== {dlm-init}: Manage your data lifecycle
+
+preview::[]
+
+The goal of {dlm} ({dlm-init}) is to enable you to automatically manage data streams
+according to your retention requirements. For example, you could use {dlm-init} to:
+
+* Ensure that data indexed in a data stream will be kept at least for the retention time you defined.
+* Older data will be deleted by {es} at a later time.
```

Comment:
This second bullet doesn't seem to be parallel with the first, and doesn't make sense following `you could use DLM to`. Maybe something like `Ensure that data older than your retention time will be deleted automatically` or something?

Reply:
Thanks! You are absolutely right!